docs/assets/diagrams/transfer/03a-confirm-payee.puml

```diff
@@ -0,0 +1,127 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+Alice -> App: The payee info is correct
+
+rnote right of App #Light
+  ""updateDoc(""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    sourceAccountId: "bob.fspa",""
```

Comment:
Does sourceAccountId come from linking? Where is it persisted in the app?

Reply:
Yes, I think for the `sourceAccountId` and `payer` (just updated) fields may need to be persisted in the app from account linking. If I understand it correctly, the user needs to choose `sourceAccountId` in the app before initiating the transaction, and the `payer` data may need to be cached by the app (or perhaps could be stored by the server as well).

Reply:
Please consider what happens if the app is uninstalled and reinstalled

Reply:
I was assuming this will be stored by the server. A list of linked accounts for an user of the app.

Reply:
Oh yeah, sorry I mean the server will also store the data, but the mobile app is also expected to cache the data locally.

Reply:
Added a note about this in the documentation